illumio/__init__.py

```diff
@@ -20,12 +20,13 @@
 from .rules import *
 from .explorer import *
 from .pce import *
+from .appgroups import *
 
 from types import ModuleType
 
 # avoid name conflicts with package modules when using
 # `from illumio import *` by excluding them here
 __all__ = [
     export for export, o in globals().items()
-        if not (export.startswith('_') or isinstance(o, ModuleType))
+    if not (export.startswith('_') or isinstance(o, ModuleType))
```

Comment:
Nit: Is this from an automated formatter? Please leave these as-is.

Reply:
I can do that but it won't be PEP8 compliant.

Reply:
![image](https://github.com/illumio/illumio-py/assets/98968308/0ad96f70-c8bf-45cc-a2f7-8e53cb65b818)

Reply:
PEP8 isn't a compliance framework, it's a style guide. The whole point is to improve readability through consistently applied rules. In this case, the unaligned version is more readable because it indicates that the condition is a filter on the `for`.

Reply:
already resolved in latest commit - i added the indent back.